test/creddit/test/client.clj

```diff
@@ -259,4 +259,15 @@
                 {:status 200 :headers {} :body reddit-response})}
              (client/users-popular creddit-client 10))
            parsed-reddit-response))
-    (is (thrown? Exception (client/users-popular creddit-client "10")))))
\ No newline at end of file
+    (is (thrown? Exception (client/users-popular creddit-client "10")))))
+
+(deftest test-frontpage
+  (testing "Retrieve named posts"
+    (is (= (with-fake-routes-in-isolation
+             {"https://www.reddit.com/by_id/.json?names=t3_7ktt45,t3_7jj5nf"
```

Comment:
This URL is different to the one your listing function is calling which is causing the tests to fail.

Should be **https://www.reddit.com/by_id/t3_7ktt45,t3_7jj5nf/.json**

Reply:
Again, oops.